build.gradle

```diff
@@ -1065,6 +1065,7 @@ project(':clients') {
     include "**/org/apache/kafka/common/security/auth/*"
     include "**/org/apache/kafka/common/security/plain/*"
     include "**/org/apache/kafka/common/security/scram/*"
+    include "**/org/apache/kafka/common/security/ssl/*"
```

Comment:
```SslEngineFactory``` document contains the reference of ```SslFactory``` so both of them should be included.

Reply:
It might not be the best approach, but we use this list here as to define the set of APIs that are considered part of the public API. Unless we want `SslFactory` to be part of the public API, maybe we should remove the doc references from `SslEngineFactory`.

Reply:
Copy that